scripts/build-examples.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-env node */
+const pkg = require('../package.json');
+const fs = require('fs-extra');
+const path = require('path');
+const glob = require('glob');
+
+const { CLIENT_APP_SDK_PC_OAUTH_CLIENT_IDS: oauthClientIds } = process.env;
+const BROWSER_FILENAME = `/${pkg.name}.js`;
+
+// Called from command line
+if (!module.parent) {
+    const { bundle } = require('yargs')
+        .option('bundle', { describe: 'minified UMD bundle file name' })
+        .demandOption(['bundle'])
```

Comment:
Should this be required or should we default it to BROWSER_FILENAME as below in `transformExampleSdkUrl`?  Or, are we assuming this is mostly a feature for builds not local dev and it's not expected users will have to build examples that often?

If we do expect users to build, maybe we should add a blurb about it in README and document the bundle optioon?

Reply:
Yeah, I think my thought here was that this would be used solely by the Jenkins job, and if someone wants to build/run examples locally then they should probably just start up the dev server, which automatically does the bundling, example transformation, and live reload. With that being said, if we do want to open it up to users, I think a default of BROWSER_FILENAME is reasonable and that we should probably allow them to customize the dist folder as well.

Reply:
Yeah, i'm good with keeping it as primarily a deploy function.  Maybe a note in the readme describing the local server and examples vs the deployed examples would be good ... but only if we have to make another commit to this PR for something less nitty.